app/presenters/tree_builder_report_reports_class.rb

```diff
@@ -5,6 +5,7 @@ class TreeBuilderReportReportsClass < TreeBuilder
 
   def x_get_tree_r_kids(object, count_only)
     objects = MiqReportResult.with_current_user_groups_and_report(object.id).to_a
```

Comment:
@ZitaNemeckova what about to use ordering direclty on query ?

`MiqReportResult.with_current_user_groups_and_report(object.id).order(:last_run_on)` (eventually `.order("last_run_on ASC")` or `.order("last_run_on ASC")`)